src/app/shared/services/broker.service.ts

```diff
@@ -33,19 +31,25 @@ export class BrokerService {
       .get(`${this.API_URL}/submissions/${submissionUuid}/spreadsheet`,
         {observe: 'response', responseType: 'blob'})
       .pipe(timeout(this.DOWNLOAD_SPREADSHEET_TIMEOUT),
-        map((res) => {
-          return this.getFileDataFromResponse(res);
-      }));
+        map(response => {
+          if (response.status === HttpStatusCode.Ok) {
+            return this.getFileDataFromResponse(response);
+          } else if (response.status === HttpStatusCode.Accepted) {
+            throwError(new Error("The spreadsheet is not yet available and still being generated."))
```

Comment:
Why is this considered an error as opposed to a normal return value?

Reply:
The accepted status means that the file is not yet available but is already being generated. The file not found status is being returned if there is no spreadsheet being generated at all.